packages/migration/src/types.ts

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import type {
+  AnimationType,
+  AnimationPanDirection,
+  AnimationZoomDirection,
+  ElementBox,
+  Attribution,
+  ResourceType,
+  Mask,
+  Border,
+  BorderRadius,
+  MediaElement,
+  VideoTrack,
+  TrimData,
+} from '@googleforcreators/types';
+
+type TextAlign = 'left' | 'center' | 'right' | 'justify';
+type TagName = 'h1' | 'h2' | 'h3' | 'p' | 'auto';
+
+export interface FontMetrics {
```

Comment:
@swissspidy I've updated this file to work back to the Story V0 type (currently unorganized and still depending on the global current types at this point).

A few things to consider
- There are some props that we know did not exist in the V0, such as the font metrics or all the trimming related data etc. However, it's not touched by the migration script at all. Should we remove these and try to track down when they were added, or is it ok to just keep them from V0 (since they're not touched by the migration script)? Thoughts?
- Same goes for the GIF element for example. It was added at some point -- is it ok to assume it existed since V0 even though it didn't?
- Since the current types are changing, I guess we should remove all dependency on the `types` packages and port all the types over to here for V0. Currently there are a few imports still in this file.

Reply:
I think it would be too much effort to track down when these were added, so should be ok to just leave it. Maybe with an explanatory comment to avoid confusion in the future.

> * Since the current types are changing, I guess we should remove all dependency on the `types` packages and port all the types over to here for V0. Currently there are a few imports still in this file.

Makes sense. `types` is ever-changing so we can't rely on it here via import.